urdf_model_state/include/urdf_model_state/model_state.h

```diff
@@ -49,6 +49,15 @@
 
 namespace urdf{
 
+//round is not defined in C++98
+//So in Visual Studio is necessary to define it
+#ifdef _MSC_VER
```

Comment:
It seems [`std::round` was implemented since VS2013](http://blogs.msdn.com/b/vcblog/archive/2013/07/19/c99-library-support-in-visual-studio-2013.aspx). I found some posts that says it's not implemented until VS2012: [1](https://social.msdn.microsoft.com/Forums/vstudio/en-US/260e04fc-dd05-4a96-8953-9c6ea1ad62fb/cant-find-stdround-in-cmath?forum=vclanguage), [2](http://stackoverflow.com/questions/21065537/where-are-round-rint-and-so-on-c-functions-in-visual-studio-2012).